content/help/ideation/index.md

```diff
@@ -3,15 +3,13 @@ title: Ideation
 category: help
 menu:
   help:
+    parent: addons
     identifier: ideation
-    weight: 5
 aliases:
 - /ideation
 ---
 {{% cloudfeature %}}
 
-## Ideation
-
 Ideation gives forum users the ability to upvote and optionally downvote discussions.
 
 ### Ideas Overview
```

Comment:
Should be h2

Reply:
Ah right. I've removed that h2 (because it's also the h1 at the top) and made all of the h3's h2's now.